spring-web/src/main/java/org/springframework/http/server/reactive/JettyCoreServerHttpResponse.java

```diff
@@ -133,8 +132,7 @@ public Mono<Void> writeWith(Path file, long position, long count) {
 		Callback.Completable callback = new Callback.Completable();
 		mono = Mono.fromFuture(callback);
 		try {
-			// TODO: Why does intellij warn about possible blocking call?
-			//       Because it can block and we want to be fully asynchronous.  Use AsynchronousFileChannel?
+			@SuppressWarnings("BlockingMethodInNonBlockingContext")
```

Comment:
I would still add a comment saying that the method can block, but it is not expected to do so for any significant time.   It could be rewritten asynchronously if it ever proves to be a problem.

Reply:
done